rplugin/python3/deoplete/sources/deoplete_jedi.py

```diff
@@ -192,11 +192,17 @@ def gather_candidates(self, context):
         if cached is None:
             wait = True
 
+        # Extra options to pass to the server.
+        options = {
```

Comment:
Use `context` instead of `options` here, and below?
What about just passing through `context` altogether?